### ISSUE TITLE
Users should be able to increase the type size in reader view [feature]

### ISSUE THREAD

**Issue by seanttaylor** [seanttaylor/kronkite#17]
Keep accessibility features in mind! Users should have a little leeway in how large the typeface in reader view appears. Maybe 1 or 2 selectable options for type-size? Not everybody has super young eyes....
